src/components/flat-table/flat-table-body-draggable/flat-table-body-draggable.spec.js

```diff
@@ -0,0 +1,241 @@
+import React from "react";
+import { mount } from "enzyme";
+import { act } from "react-dom/test-utils";
+import { render } from "react-dom";
+import {
+  FlatTable,
+  FlatTableRow,
+  FlatTableCell,
+  FlatTableBodyDraggable,
+} from "..";
+
+describe("Draggable Table", () => {
+  let wrapper;
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should return an array with id's", () => {
+    const getOrder = jest.fn();
+    wrapper = mount(
+      <FlatTable>
+        <FlatTableBodyDraggable getOrder={getOrder}>
+          <FlatTableRow key={0} id={0} index={0}>
+            <FlatTableCell>UK</FlatTableCell>
+          </FlatTableRow>
+          <FlatTableRow key={1} id={1} index={1}>
+            <FlatTableCell>Germany</FlatTableCell>
+          </FlatTableRow>
+        </FlatTableBodyDraggable>
+      </FlatTable>
+    );
+    wrapper.find("DropTarget").at(0).props().getOrder();
+    expect(getOrder).toHaveBeenCalledWith([0, 1]);
+  });
+
+  it("should return an array if getOrder is not passed to the component", () => {
+    const getOrder = jest.fn();
+    wrapper = mount(
+      <FlatTable>
+        <FlatTableBodyDraggable>
+          <FlatTableRow key={0} id={0} index={0}>
+            <FlatTableCell>UK</FlatTableCell>
+          </FlatTableRow>
+          <FlatTableRow key={1} id={1} index={1}>
+            <FlatTableCell>Germany</FlatTableCell>
+          </FlatTableRow>
+        </FlatTableBodyDraggable>
+      </FlatTable>
+    );
+    wrapper.find("DropTarget").at(0).props().getOrder();
+    expect(getOrder).not.toHaveBeenCalledWith();
+  });
+
+  it("should render correct", () => {
+    expect(wrapper.exists()).toBe(true);
+  });
+});
+
+describe("Draggable Table", () => {
```

Comment:
We have two `describe` blocks with the same name, this should be "Draggable Table with sub rows" or something